### ISSUE TITLE
Add vision stuff

### ISSUE THREAD

**Issue by myrrhkhan** [Team612/612-2020#2]
This is a test issue. Disregard. I'm testing to see if we can use the issues feature more often with our 2022 bot.

A) robot is bad
B) needs more vision

**Comment by myrrhkhan** [comment]
#1 might be a remedy
(this whole thing is a test. disregard)

**Comment by calvin-godfrey** [comment]
Issues are incredibly useful. Check how how we used them in the [2017 repo](https://github.com/Team612/612-2017/issues).  You can make tags for issues to show priority and what it's about. For example, we had tags like "All hands on deck" if it was a major issue, or "help wanted" if it wasn't assigned to anyone. You can also assign issues to people so that you can easily see who is working on what.

**Comment by myrrhkhan** [comment]
Yeah, I’ve seen that before in other repos. GitHub has an interesting beta feature that puts issues into a Trello-like board that might be useful. Thanks for sharing!!

edit: Trello has its own GitHub plugin as well, so that’s great.
